internals/app/services.go

```diff
@@ -125,6 +130,36 @@ func initEmailSender() {
 	password := viper.GetString("SMTP_PASSWORD")
 	host := viper.GetString("SMTP_HOST")
 	port := viper.GetString("SMTP_PORT")
-	email.InitSender(username,password,host,port)
+	email.InitSender(username, password, host, port)
 
 }
+
+func initOidcAuthentication() {
+	authenticationMode := viper.GetString("AUTHENTICATION_MODE")
+
+	if authenticationMode == "OIDC" {
+		oidcIssuerUrl := viper.GetString("AUTHENTICATION_OIDC_ISSUER_URL")
+		oidcClientID := viper.GetString("AUTHENTICATION_OIDC_CLIENT_ID")
+		oidcClientSecret := viper.GetString("AUTHENTICATION_OIDC_CLIENT_SECRET")
+		oidcRedirectURL := viper.GetString("AUTHENTICATION_OIDC_REDIRECT_URL")
+		scopesString := viper.GetString("AUTHENTICATION_OIDC_SCOPES")
+		oidcScopes := strings.Split(scopesString, ",")
+
+		if oidcIssuerUrl == "" || oidcClientID == "" || oidcClientSecret == "" || oidcRedirectURL == "" || scopesString == "" {
+			zap.L().Error("Missing OIDC configuration")
```

Comment:
When this error occurs, what auth mode is set ?

Reply:
I made sure to display an error in the console indicating that information is missing to initialize the OIDC client, and I automatically switch to basic authentication."

Reply:
where is it set ?